### DISCUSSION TITLE
how are the next/prev buttons generated?

### DISCUSSION THREAD

**Discussion by mluerig** [pradyunsg/furo#329] [Questions] [answered]
can anyone tell me how those next/prev buttons at the bottom of the page are generated? I seem to have lost them and I don't know why...

**Comment by pradyunsg** [comment]
You've likely tried to override the footer. Look for a `block footer` in your documentation sources. :)

**Comment by mluerig** [comment, marked as answer]
thanks. it solved itself before I could trace it
